### PR TITLE
Docs: Fix typos and grammar

### PR DESCRIPTION
**Pull request by liyier90**
In addition to typo and grammar changes:

- Change `Node()` initialization in tutorials to use keyword arguments directly instead of `**dict` operator

### CODE REVIEW

File: docs/source/tutorials/05_calling_peekingduck_in_python.rst [hunk 1]
```diff
@@ -239,23 +238,25 @@ Lines 17 - 23: The debugging output showing the frame number and the confidence
...
       <https://www.tensorflow.org/api_docs/python/tf/keras/utils/image_dataset_from_directory>`_
       (available in ``tensorflow>=2.3.0``),
-    * External models (not implemented as PeekingDuck nodes) such as `easyocr
+    * External packages (not implemented as PeekingDuck nodes) such as `easyocr
       <https://pypi.org/project/easyocr/>`_, and
     * Visualization packages such as `matplotlib <https://pypi.org/project/matplotlib/>`_.
 
-The notebook corresponding to this tutorial can be found in the `notebooks <https://github.com/aimakerspace/PeekingDuck/tree/dev/notebooks>`_ folder of the
-PeekingDuck repository and is also available at a `Colab notebook <https://colab.research.google.com/drive/1NwQKrnY_3ia2mBEaUinkvUqbrjjT3ssq#scrollTo=l2MCyh5Hgp5O>`_.
+The notebook corresponding to this tutorial, ``demo_import_peekingduck.ipynb``, can be found in the
```
Comment (leeping-ng) [inline docs/source/tutorials/05_calling_peekingduck_in_python.rst:256]: For consistency, can we rename this notebook and its heading?

Comment (liyier90) [reply docs/source/tutorials/05_calling_peekingduck_in_python.rst]: updated to `calling_peekingduck_in_python.ipynb`

File: notebooks/calling_peekingduck_in_python.ipynb [hunk 2]
```diff
@@ -184,7 +184,7 @@
     "This grants us the flexibility of inspecting the data for debugging or interacting with the data to integrate the inference loop external processes.\n",
     "\n",
     "We have also defined a utility function for retrieving the image region of the license plate with a highest confidence score to improve code clarity.\n",
-    "For more information on how to convert between bounding box and image coordinates, please refer to the [\"Bounding Box vs Image Coordinates\"](https://peekingduck.readthedocs.io/en/docs-v1.2/tutorials/01_hello_cv.html#bounding-box-vs-image-coordinates) section in our tutorials."
+    "For more information on how to convert between bounding box and image coordinates, please refer to the [\"Bounding Box vs Image Coordinates\"](https://peekingduck.readthedocs.io/en/stable/tutorials/01_hello_cv.html#bounding-box-vs-image-coordinates) section in our tutorials."
    ]
   },
   {
```
Comment (leeping-ng) [inline notebooks/calling_peekingduck_in_python.ipynb:190]: The title of this notebook is still `Import PeekingDuck as a Module Demo`, best to update as well for consistency.

Comment (liyier90) [reply notebooks/calling_peekingduck_in_python.ipynb]: Updated, changed the filename/title in the Colab notebook as well

**Pull request merged**